<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="settingsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="rotationBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Rotation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QDial" name="RotationDialX">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the X-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="RotationDialY">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the Y-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="RotationDialZ">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the Z-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="ResetRotationButton">
            <property name="text">
             <string>Reset Rotation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="scalingBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Scaling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSlider" name="ScaleSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetScaleButton">
            <property name="text">
             <string>Reset Scaling</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="textureBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>gBuffer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="AllButton">
            <property name="text">
             <string>&amp;All</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ColorButton">
            <property name="text">
             <string>&amp;Color</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="NormalsButton">
            <property name="text">
             <string>&amp;Normals</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="PositionButton">
            <property name="text">
             <string>&amp;Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="DepthButton">
            <property name="text">
             <string>&amp;Depth</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>60</height>
          </size>
         </property>
         <property name="title">
          <string>Performance</string>
         </property>
         <widget class="QPushButton" name="perfAnalysisButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>181</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string>Run performance test</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="settingBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>102</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QCheckBox" name="enableSun">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>104</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Enable sun</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="enableLights">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>91</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Enable lights</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="animate">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>86</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Animate</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="reflectionBox">
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Reflection</string>
         </property>
         <widget class="QCheckBox" name="ambientButton">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>30</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Ambient</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="diffuseButton">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>60</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Diffuse</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="specularButton">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>90</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Specular</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="mainView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
